### PR TITLE
Return ouder if `burgerservicenummer` is present

### PR DESCRIPTION
**Pull request by roel-devries**
see documentation in code

### CODE REVIEW

File: src/gobstuf/stuf/brp/response_mapping.py [hunk 1]
```diff
@@ -757,7 +757,12 @@ def filter(self, mapped_object: dict, **kwargs):
             # Compares only the same number of characters as the given string, to match precision
             return None
         elif not naam.get('geslachtsnaam') and not naam.get('voornamen') and not mapped_object.get('geboorte'):
```
Comment (JKoetsier) [inline src/gobstuf/stuf/brp/response_mapping.py:759]: Maybe replace with something like `elif not any([name.get(attr)) for attr in [....])`? Bit easier to read

**Review by JKoetsier — APPROVED** [review]
Approve with a small comment on an unchanged line of code. Not part of your changes, so no dealbreaker here.

File: src/gobstuf/stuf/brp/response_mapping.py (hunk 1 above)
Comment (JKoetsier) [reply src/gobstuf/stuf/brp/response_mapping.py:759]: Now that I opened the rest of the code above, there's some more room for improvement here. All the `mapped_object.get()` calls with or without negations with all the checks are a bit hard to read.

I'd replace the (el)if expressions for short function calls, where the function names are descriptive of the check. Also removes the need for the extensive comments, makes the code much easier to read.

Again; no need to change that in this PR, but something to consider in the future.

Comment (roel-devries) [reply src/gobstuf/stuf/brp/response_mapping.py:759]: yup, I agree, also trying to omit negations if possible

**Pull request merged**